demo/EXPLOR3R/auto-drive.py

```diff
@@ -64,9 +66,9 @@ def backup():
     Back away from an obstacle.
     """
 
-    # Sound backup alarm.
+    # Sound backup alarm. 
     Sound.tone([(1000, 500, 500)] * 3)
-
+    
```

Comment:
@DrClick I think this looks good aside from the whitespace. If you can remove that (both here and two lines above it) I'd say it's ready!